Realm/RealmList.cpp

```diff
@@ -44,19 +44,32 @@ extern DatabaseType LoginDatabase;
 static const RealmBuildInfo ExpectedRealmdClientBuilds[] =
 {
     // highest supported build, also auto accept all above for simplify future supported builds testing
-    {40000, 9, 0, 0, ' '},  // SHADOWLANDS
-    {35662, 8, 3, 7, ' '},  // BFA
-    {26972, 7, 3, 5, ' '},  // Legion
-    {21742, 6, 2, 4, ' '},  // WOD
-    {18414, 5, 4, 8, ' '},  // MOP
-    {18273, 5, 4, 8, ' '},  // MOP
-    {15595, 4, 3, 4, ' '},  // CATA
-    {12340, 3, 3, 5, 'a'},  // WOTLK
-    {8606,  2, 4, 3, ' '},  // TBC
-    {6141,  1, 12, 3, ' '}, // Vanilla - Chinese
-    {6005,  1, 12, 2, ' '}, // Vanilla - Spanish
-    {5875,  1, 12, 1, ' '}, // Vanilla
-    {0,     0, 0, 0, ' '}                                   // terminator
+    { 40000, 9, 0, 0, ' ', { { } }, { { } } }, // shadow lands
+    { 35662, 8, 3, 4, ' ', { { } }, { { } } }, // bfa
+    { 26972, 7, 3, 5, ' ', { { } }, { { } } }, // legion
+    { 21742, 6, 2, 4, ' ', { { } }, { { } } }, // wod
```

Comment:
To my knowledge, clients post MoP don't support legacy authentication. they only work with bnet server, so what's the point of defining them here? They can never connect to realmd, as they don't support that protocol.

Reply:
Kind of what I thought as well.